Library/Homebrew/software_spec.rb

```diff
@@ -64,6 +66,11 @@ def url(val = nil, specs = {})
     dependency_collector.add(@resource)
   end
 
+  def gpg(val = nil)
```

Comment:
Can you use attr_rw here? Maybe not but figure I'd ask.

Also, I think this contradicts the `def_delegators` above.